tests/integration/local/start_api/test_start_api.py

```diff
@@ -19,6 +19,13 @@
 from ..invoke.layer_utils import LayerUtils
 
 
+# class TestInvalidTempate(StartApiIntegBaseClass):
```

Comment:
nit: remove the lines if they are not needed

Reply:
Oops, didn't mean to commit this